go/coordinator/internal/metastore/db/dao/segment.go

```diff
@@ -165,20 +165,23 @@ func generateSegmentUpdatesWithoutID(in *dbmodel.UpdateSegment) map[string]inter
 		}
 	}
 
-	if in.ResetCollection {
```

Comment:
Why comment out this code?

Reply:
I don't see why we need to set collection id to null. And it is not possible to do it because collection_id is now primary key of segment